library/Icinga/File/Csv.php

```diff
@@ -37,7 +37,7 @@ public function __toString()
             }
             $out = array();
             foreach ($row as & $val) {
-                $out[] = '"' . ($val ? str_replace('"', '""', $val) : '') . '"';
+                $out[] = '"' . ($val === '0' ? '0' : ($val ? str_replace('"', '""', $val) : '')) . '"';
```

Comment:
Just in case it's really zero ;)

```suggestion
                $out[] = '"' . ($val == '0' ? '0' : ($val ? str_replace('"', '""', $val) : '')) . '"';
```